packages/paste-website/src/pages/icons/components.mdx

```diff
@@ -0,0 +1,139 @@
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Button} from '@twilio-paste/button';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {LoadingIcon} from '@twilio-paste/icons/esm/LoadingIcon';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {Grid} from '../../components/grid';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/icons/components/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+Paste icons are distributed as both SVG files and React components. Our base icon size is set to 24px.
+
+## Implementation
+
+### Why SVG?
+
+PNGs and JPGs are created using thousands of tiny pixels. SVGs are made out of geometric
+primitives such as points, lines, and basic shapes. Because of this attribute, SVGs can
+be resized without any loss of quality to the image. We can also write code to change SVG
+attributes in realtime. These characteristics are ideal for icons since we usually want to
+customize their size or color. To do this with PNGs or JPGs we would need to create a
+separate file for each variation.
+
+### Why React components?
+
+If you're using any Paste component, you're familiar with how to install and use a React
+component. Our icon components have accessibility baked in. They also use our tokens for
+sizing and coloring. With icon components, you can import only the icons your project needs.
+
+For more reading about icon components, please check our [rationale document](https://github.com/twilio-labs/svg-to-react/blob/master/RATIONALE.md).
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/icons
+```
+
+#### Usage
+
+Icons have to be imported individually. This import style was decided for important performance benefits. As our icon
+set grows to contain over 100 icons, we don't want projects that use only a few icons to be burdened by
+that large of a payload.
+
+```jsx
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+
+<Button variant="secondary" size="small" onClick={() => {}}>
+  <PlusIcon decorative={true} />
+  Add to cart
+</Button>;
+```
+
+### Examples
+
+#### Adapting the icon's color
+
+Since we didn't pass a color to the `LoadingIcon` directly, it inherited the color from the wrapping `Text` component.
+The `Box` is used for alignment.
+
+<LivePreview scope={{Box, Text, LoadingIcon}} language="jsx">
+  {`<Text textColor="colorTextInverse">
+  <Box display="flex" alignItems="center" backgroundColor="colorBackgroundInverse" padding="space30" borderRadius="borderRadius20">
+    <LoadingIcon decorative={true}  /> 
+    Now loading
+  </Box>
+</Text>`}
```

Comment:
I don't love this example as I never really envisaged Text wrapping Box. My end goal for this kind of example is to have something like:

```jsx
<Box backgroundColor="colorBackgroundInverse" padding="space30" borderRadius="borderRadius20">
  <Text textColor="colorTextInverse">
    <MediaObject>
      <MediaFigure>
        <LoadingIcon decorative={true} />
      </MediaFigure>
      <MediaBody>
        Now loading
      </MediaBody>
    </MediaObject>
  </Text>
</Box>
```

But it's interesting how you approached it as just an observation